tests/sentry/api/endpoints/test_project_user_reports.py

```diff
@@ -209,18 +209,17 @@ def test_with_dsn_auth_invalid_project(self):
 
     def test_already_present(self):
         self.login_as(user=self.user)
-
         project = self.create_project()
-        group = self.create_group(project=project)
         environment = self.make_environment(project)
         event = self.create_event(
-            group=group,
-            tags={'environment': environment.name},
-            datetime=timezone.now(),
+            data={
```

Comment:
Is this intentionally still using `create_event`?

Reply:
No, I was just very lazy with porting. Will fix though, this seems like a half-way port